core/commonMain/src/kotlinx/serialization/descriptors/SerialDescriptor.kt

```diff
@@ -128,31 +128,44 @@ import kotlinx.serialization.encoding.*
  * }
  * ```
  *
- * For a classes that are represented as a single primitive value, [PrimitiveSerialDescriptor] builder function can be used instead.
+ * For classes that are represented as a single primitive value, [PrimitiveSerialDescriptor] builder function can be used instead.
  *
  * ### Consistency violations
  * An implementation of [SerialDescriptor] should be consistent with the implementation of the corresponding [KSerializer].
- * Yet it is not type-checked statically, thus making it possible to declare a non-consistent implementations of descriptor and serializer.
- * In such cases, the behaviour of an underlying format is unspecified and may lead to both runtime errors and encoding of
+ * Yet it is not type-checked statically, thus making it possible to declare a non-consistent implementation of descriptor and serializer.
+ * In such cases, the behavior of an underlying format is unspecified and may lead to both runtime errors and encoding of
```

Comment:
We use both options, is this change definitely necessary?

Reply:
IIRC general style guideline dictates the usage of American English, so all spelling should be converted to this one eventually